doc/FAQ.rst

```diff
@@ -116,6 +125,20 @@ that let you refer to the object easily in the code, and then you can
 set the plot title directly, using the plot option
 ``title_format="my new title"``.
 
+For frequent cases you can define a dictionary of aliases, with short,
+Pythonic names for complicated typeset strings:
+
+```
+al = hv.util.Aliases(Spectrum='Frequency spectrum', 
+                     Water='$H_2O$',
+                     Glucose='$C_6H_{12}O_6$')
+
+(hv.Image(np.random.rand(10,10), group=al.Spectrum, label=al.Glucose) +
+ hv.Image(np.random.rand(10,10), group=al.Spectrum, label=al.Water))
+```
```

Comment:
I think documenting this in the FAQ isn't too bad in the end. It is probably things like a collection of  ``final_hooks`` examples that are hard to find a place for...